dev_docs/tutorials/endpoints.mdx

```diff
@@ -0,0 +1,78 @@
+---
+id: kibDevTutorialServerEndpoint
+slug: /kibana-dev-docs/tutorials/registering-endpoints
+title: Registering and accessing an endpoint
+summary: Learn how to register a new endpoint and access it
+date: 2021-10-05
+tags: ['kibana', 'dev', 'architecture', 'tutorials']
+---
+
+## Registering an endpoint
```

Comment:
Might be worth linking to the example plugin - https://demo.kibana.dev/8.0/app/routingExample?